### PR TITLE
Update font with new version

### PR DESCRIPTION
**Pull request by osaf96**
replaced font files and updated the font.css file

### CODE REVIEW

File: public/fonts/fonts.css [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-    src: url("ISOv0.9-Medium.woff2") format('woff2');
+    src: url("VCTRMonov0.12-Medium.woff2") format('woff2');
     font-style: normal;
     font-weight: 600;
   }
   @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-Medium.woff2") format('woff2');
+    src: url("VCTRMonov0.12-MediumItalic.woff2") format('woff2');
     font-style: normal;
```
Comment (vnugent) [inline public/fonts/fonts.css]: Can you change `font-style: normal` to `font-style: italic`?  (since we already have font-style: normal for 600 above)

**Comment by vnugent** [comment]
can you also add **italic** for Thin, Light, Bold and Black (they're not available before)

```
   // thin TBD

  @font-face {
    font-family: "ISO";
    src: url("VCTRMonov0.12-LightItalic.woff") format('woff2');
    font-style: italic;
    font-weight: 200;
  }

  @font-face {
    font-family: "ISO";
    src: url("VCTRMonov0.12-BoldItalic.woff2") format('woff2');
    font-style: italic;
    font-weight: 700;
  }

// black TBD

```

**Review by vnugent — CHANGES_REQUESTED** [review]

**Comment by vnugent** [comment]
@all-contributors bot please add @osaf96 as a contributor for code

File: public/fonts/fonts.css (hunk 1 above)
Comment (osaf96) [reply public/fonts/fonts.css]: sure

**Pull request merged**